languages/rust/reference/merge-strategy.md

```diff
@@ -0,0 +1,34 @@
+# merging strategy
+
+The Rust track maintainers [have decided](https://github.com/exercism/v3/discussions/1725#discussion-7438) to write down our merging strategy to communicate expectations and encourage quality contributions.
+
+## general principles
+
+- we strive to accept _all_ contributions
+  - We'll bring contributions to a high standard of quality through collaborative, iterative review.
+  - We adopt a pessimistic merging approach. Think of maintainer reviews like the Rust compiler's feedback. The amount of feedback can feel overwhelming and frustrating. But we'll work with you to get it over the finish line! So keep up the good work.
+- One other maintainer should review a contributor's pull request.
+- At least one other maintainer should approve a pull request before merging it into the development branch.
+
+## pull request workflow
+
+### definitions
+
+| Term               |                                                                                               Meaning |
+| ------------------ | ----------------------------------------------------------------------------------------------------: |
+| Contributor        |                                                                 Anyone adding a new concept exercise. |
+| Maintainer         |         Anyone who belongs to the [@exercism/rust team](https://github.com/orgs/exercism/teams/rust). |
+| development branch | The [v3](https://github.com/exercism/v3) repository's default development branch. Currently `master`. |
+
+### the workflow
+
+1. Contributors start by opening a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) with their changes
+   1. Our track follows broader exercism community here. Using the draft feature signals the work is ready for high-level review.
```

Comment:
```suggestion
1. Contributors start by opening a [draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/) or a complete pull request with their changes
   1. Using the draft feature signals the work is ready for high-level review, but that details may currently be unimplemented, incomplete, or wrong.
   1. After an unspecified number of iterations of steps 2-3, the contributor marks the PR as ready for review.
```

Clarifying what distinguishes a draft from a full PR, and that contributors do not have to necessarily begin with a draft. Removed the bit about following community norms: that's both assumed, and kind of irrelevant.